crates/runtime/src/backtrace/aarch64.rs

```diff
@@ -0,0 +1,19 @@
+pub unsafe fn get_next_older_pc_from_fp(fp: usize) -> usize {
+    *(fp as *mut usize).offset(1)
+}
+
+pub unsafe fn get_next_older_fp_from_fp(fp: usize) -> usize {
+    *(fp as *mut usize)
+}
+
+pub fn reached_entry_sp(fp: usize, first_wasm_sp: usize) -> bool {
+    fp == first_wasm_sp - 16
+}
```

Comment:
Hi @akirilov-arm, would you mind suggesting wording for some comments for these functions similar to those in the `x86_64.rs` file just below? I'm not familiar enough to add the docs that Alex requested in review myself. Thanks!

Reply:
Thanks, I saw, just haven't gotten around to adding them yet. Appreciated!